vmdb/app/helpers/provider_foreman_helper.rb

```diff
@@ -1,30 +1,126 @@
 module ProviderForemanHelper
   def textual_group_properties
-    items = %w(hostname configuration_profile configuration_profile_desc provider provider_url zone)
+    items = %w(hostname ipmi_present ipaddress mac_address configuration_profile_desc provider_name zone)
```

Comment:
since this is a static list. Is there a reason to not just put textual_ into the static list?

```ruby
%w(textual_hostname textual_ipmi_present textual_ipaddress textual_mac_address
   textual_configuration_profile_desc textual_provider_name textual_zone).flat_map { |m| send(m) }.compact
```